server-common/src/main/java/com/datastrato/gravitino/server/web/CorsFilterHelper.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.server.web;
+
+import org.eclipse.jetty.servlet.FilterHolder;
+import org.eclipse.jetty.servlets.CrossOriginFilter;
+
+class CorsFilterHelper {
+
+  static final String PREFLIGHT_MAX_AGE = "preflightMaxAge";
+
+  private CorsFilterHelper() {}
+
+  public static FilterHolder createCorsFilterHolder(JettyServerConfig config) {
+    FilterHolder filterHolder = new FilterHolder();
+    filterHolder.setClassName(CrossOriginFilter.class.getName());
+    filterHolder.setInitParameter(
+        JettyServerConfig.ALLOWED_ORIGINS.getKey(), config.getAllowedOrigins());
```

Comment:
Will this cors filter check the validity of the configuration? For example, if the configuration is not legal or have additional/trailing whitespace.

Reply:
No, this cors filter check less things. Only check int type whether can be parsed to Integer.